.github/CONTRIBUTNG.md

```diff
@@ -0,0 +1,31 @@
+# Contributing
+
+Contributions are welcome, and they are greatly appreciated! Every little bit helps, and credit will always be given.
+
+Please read the PAVICS [Developer Documentation](https://pavics-sdi.readthedocs.io/en/latest/dev/index.html) to get started.
```

Comment:
Ouch, instructions in this https://pavics-sdi.readthedocs.io/en/latest/dev/index.html is not complete with respect to this stack and even link to the old https://github.com/Ouranosinc/PAVICS.git.

I suggest this link instead https://birdhouse-deploy.readthedocs.io/en/latest/.

Or mention both links since the pavics-sdi one is more general and this birdhouse-deploy is very specific to this stack (all the options/components, how to activate them ...)